dockerfiles/splash/install-python-splash-deps.sh

```diff
@@ -5,7 +5,7 @@ install_python_deps () {
     # Install python-level dependencies.
     ${_PYTHON} -m pip install -U pip setuptools six && \
     ${_PYTHON} -m pip install \
-        qt5reactor==0.5 \
+        qt${QT_MAJOR_VERSION}reactor==0.5 \
```

Comment:
this is a specific pypi package, I'm not sure they're going to change the package name